packages/contracts/script/utils/exporter.js

```diff
@@ -99,7 +98,9 @@ async function clearExistingData(client) {
 
 (async () => {
   try {
-    await exportAddress(loadConfiguration());
+    const chainId = Number(process.argv[2]);
```

Comment:
Now accepts the Chain Id, passed in from the invoker (`make`, which can use a `cast chainid ${RPC_URL}`). This means the value is not in configuration and the scripts can be sensitive to the last invocation only.